app/services/datastream_importer.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+# This class imports pre-parsed datastream info into the compliance DB
+class DatastreamImporter
+  def initialize(datastream_filenames)
+    @datastreams = datastream_filenames.map do |datastream_filename|
+      OpenscapParser::Datastream.new(File.read(datastream_filename))
+    end
+  end
+
+  def import!
+    @datastreams.map do |ds|
+      Xccdf::Benchmark.transaction do
+        benchmark = save_benchmark(benchmark: ds.benchmark)
+        profiles = save_profiles(benchmark: benchmark, profiles: ds.profiles)
+        # save_rules(ds.rules)
+      end
+    end
+  end
+
+  private
+
+  def save_benchmark(benchmark: OpenscapParser::Benchmark.new)
+    benchmark = Xccdf::Benchmark.find_or_initialize_by(
+      ref_id: benchmark.id,
+      version: benchmark.version
+    )
+
+    benchmark.update(
+      title: benchmark.title,
+      description: benchmark.description,
+    )
+
+    benchmark
+  end
+
+  def save_profiles(benchmark: Xccdf::Benchmark.new, profiles: [])
+    profiles.map do |profile|
+      profile = Profile.find_or_initialize_by(
+        ref_id: profile.id,
+        name: profile.title,
+        account_id: nil,
```

Comment:
I'm a bit confused by this method. Currently profiles are created at import time, and the account ID is assigned to them at the same time. We could set the benchmark_id to those profiles and that would be it. 

Is this meant to import just the canonical profiles that are in the SSG? If so, we have to introduce the `parent_id` in this PR already.